src/Microsoft.TestPlatform.Extensions.TrxLogger/TrxLogger.cs

```diff
@@ -443,14 +444,27 @@ private void DeriveTrxFilePath()
         {
             if (this.parametersDictionary != null)
             {
-                var isLogFileNameParameterExists = this.parametersDictionary.TryGetValue(TrxLoggerConstants.LogFileNameKey, out string logFileNameValue);
-                if (isLogFileNameParameterExists && !string.IsNullOrWhiteSpace(logFileNameValue))
+                var isLogFilePrefixParameterExists = this.parametersDictionary.TryGetValue(TrxLoggerConstants.LogFilePrefixKey, out string logFilePrefixValue);
```

Comment:
Can someone give both logFilePrefix and LogFileName ?

Reply:
We will throw in logger initialize in such case